app/javascript/Pages/components/ButtonFilter.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./ButtonChip.scss";
+
+const ButtonFilter = ({ props, children }) => {
```

Comment:
**issue (blocking)**: proptypes are missing for this component, and as mentioned before you can set the defaultprops so you don't have to pass "params" and "key" as empty strings